### ISSUE TITLE
Validation Failed

### ISSUE THREAD

**Issue by hrishikesh-k** [suziwen/gatsby-plugin-gitalk#1]
Hello, I'm trying to get this plugin to work.

Here's my `gatsby-config.js`:

```
let gitalkConfig =
  {
    clientID: `value from OAuth application settings`,
    clientSecret: `value from OAuth application settings`,
    repo: `.git link of my repo given by GitHub`,
    owner: `my GitHub username`,
    admin: [`my GitHub username`],
    language: `en`
  }
module.exports =
  {
    plugins: [
      {
        resolve: `gatsby-plugin-gitalk`,
          options:
            {
              config: gitalkConfig
            }
      }]
  }
```

In my `post.js`, I've this:

```
import Gitalk from 'gatsby-plugin-gitalk'
const Post = ( ) =>
  {
    let gitalkConfig = {id: path, title: title} //path and title give me the respective details in my case
    return ({<Gitalk options = {gitalkConfig}/>})
  }
export default Post
```

However, on the page, once it's rendered, I get this

`Error: Validation Failed. The listed users and repositories cannot be searched either because the resources do not exist or you do not have permission to view them.`

I've no idea what am I doing wrong. Is it because I'm just viewing this in development and my code isn't on GitHub yet? Seems unlikely though.

**Comment by suziwen** [comment]
Check your repo is public
```
 repo: `.git link of my repo given by GitHub`, // this repo need exist and public.
```

**Comment by hrishikesh-k** [comment]
Yeah, I was just coming here to update it. I thought, it has to be my repo link ending with `.git`, but, I looked on Gitalk demo page in  the Network tab in Chrome developer tools and found out that I don't have to use my repo link, instead just the name. Sorry for the stupid confusion. Though, in the process of testing and making changes, I ended up using the official Gitalk npm package instead of this plugin, I'm sure this would work too.
